src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -0,0 +1,256 @@
+﻿namespace NServiceBus.Core.Tests.Msmq
+{
+    using System;
+    using System.Messaging;
+    using System.Security.Principal;
+    using NUnit.Framework;
+    using Transport;
+
+    [TestFixture]
+    public class MsmqQueueCreatorTests
```

Comment:
Do these tests assume local admin right or anything like that? If yes how about asserting that in the setup call?